backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepository.java

```diff
@@ -86,6 +86,22 @@ public DemoOktaRepository() {
     return Optional.of(orgRoles);
   }
 
+  public void reprovisionUser(IdentityAttributes userIdentity) {
+    final String username = userIdentity.getUsername();
+    if (!usernameOrgRolesMap.containsKey(username)) {
+      throw new IllegalGraphqlArgumentException(
+          "Cannot reprovision Okta user with unrecognized username");
+    }
+    if (!inactiveUsernames.contains(username)) {
+      throw new IllegalGraphqlArgumentException(
+          "Cannot reprovision user in unsupported state: (not deleted)");
```

Comment:
feels like the error message "A user with this email address already exists." could be reused here

Reply:
This error isn't very relevant in the `Demo` repo, but in `Live` it contains the incorrect state of the Okta user (since we expect them to be `SUSPENDED` when they are deleted through our API).  There has been some manual manipulation of user state through the Okta web interface and there is a possibility that we run into one of those users, which is when this error would surface.